src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -36,5 +34,25 @@ private string ValidateReporter(string reporter)
             }
             return reporter;
         }
+
+        private LogEventLevel ValidateLogLevel(string levelText)
+        {
+            switch (levelText?.ToLower() ?? "")
+            {
+                case "error":
+                case "":
+                    return LogEventLevel.Error;
+                case "warning":
+                    return LogEventLevel.Warning;
+                case "info":
+                    return LogEventLevel.Information;
+                case "debug":
+                    return LogEventLevel.Debug;
+                case "trace":
+                    return LogEventLevel.Verbose;
```

Comment:
This enum level is only used internally right? If that's the case, we should rename `Verbose` to `Trace` because this is a bit confusing.

Reply:
Actually, this enum comes from Serilog, so it can't be renamed.